healthtools/tests/test_scrapers.py

```diff
@@ -196,58 +157,67 @@ def test_health_facilities_scraper_gets_token(self):
             data = my_file.read()
         upload_response = self.clinical_officers_scraper.elasticsearch_index(json.loads(data))
         delete_response = self.clinical_officers_scraper.elasticsearch_delete_docs()
         self.assertEqual(len(upload_response["items"]), delete_response["deleted"])
 
     def test_delete_health_facilities_documents_from_elasticsearch(self):
-        with open(self.TEST_DIR + "/dummy_files/health_facilities.json", "r") as my_file:
+        with open(self.TEST_DIR + "dummy_files/health_facilities.json", "r") as my_file:
             data = my_file.read()
         upload_response = self.health_facilities_scraper.elasticsearch_index(json.loads(data))
         delete_response = self.health_facilities_scraper.elasticsearch_delete_docs()
         self.assertEqual(len(upload_response["items"]), delete_response["deleted"])
 
     def test_delete_nhif_inpatient_documents_from_elasticsearch(self):
-        with open(self.TEST_DIR + "/dummy_files/nhif_inpatient.json", "r") as my_file:
+        with open(self.TEST_DIR + "dummy_files/nhif_inpatient.json", "r") as my_file:
             data = my_file.read()
         upload_response = self.nhif_inpatient_scraper.elasticsearch_index(json.loads(data))
         delete_response = self.nhif_inpatient_scraper.elasticsearch_delete_docs()
         self.assertEqual(len(upload_response["items"]), delete_response["deleted"])
 
     def test_delete_nhif_outpatient_documents_from_elasticsearch(self):
-        with open(self.TEST_DIR + "/dummy_files/nhif_outpatient.json", "r") as my_file:
+        with open(self.TEST_DIR + "dummy_files/nhif_outpatient.json", "r") as my_file:
             data = my_file.read()
         upload_response = self.nhif_outpatient_scraper.elasticsearch_index(json.loads(data))
         delete_response = self.nhif_outpatient_scraper.elasticsearch_delete_docs()
         self.assertEqual(len(upload_response["items"]), delete_response["deleted"])
 
     def test_delete_nhif_outpatient_cs_documents_from_elasticsearch(self):
-        with open(self.TEST_DIR + "/dummy_files/nhif_outpatient_cs.json", "r") as my_file:
+        with open(self.TEST_DIR + "dummy_files/nhif_outpatient_cs.json", "r") as my_file:
             data = my_file.read()
         upload_response = self.nhif_outpatient_cs_scraper.elasticsearch_index(json.loads(data))
         delete_response = self.nhif_outpatient_cs_scraper.elasticsearch_delete_docs()
         self.assertEqual(len(upload_response["items"]), delete_response["deleted"])
+
+    def test_local_data_directory_or_s3_bucket_provided_exists(self):
+        if not AWS["s3_bucket"]:
+            self.assertTrue(os.path.exists(DATA_DIR))
+        else:
+            key = DATA_DIR + self.doctors_scraper.data_key
+            response = self.doctors_scraper.s3.get_object(Bucket=AWS["s3_bucket"], Key=key)
+            self.assertEqual(response["ResponseMetadata"]["HTTPStatusCode"], 200)
```

Comment:
@DavidLemayian this test becomes necessary when we add the creating objects method don't you think? The test will ensure that the provided bucket in the config is existent?
The methods will only create bucket objects but the bucket has to be created outside the scraper.